Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -48,3 +48,11 @@
       map: ["enum.WiresVisualLayers.MaintenancePanel"]
   - type: AccessReader
     access: [["Mail"]]
+
+- type: entity
```

Comment:
put in deltav namespace or add markers

Reply:
its already in dv...? Resources/Prototypes/DeltaV/Entities/Structures/Machines/vending_machines.yml ??